apis/python/tests/test_experiment_query.py

```diff
@@ -675,6 +675,21 @@ def test_experiment_query_column_names(soma_experiment):
         assert set(ad.obs.keys()) == {"soma_joinid", "label"}
         assert set(ad.var.keys()) == {"soma_joinid", "label"}
 
+@pytest.mark.parametrize("n_obs,n_vars", [(1001, 99)])
+def test_experiment_query_mp_disjoint_arrow_coords(soma_experiment):  
+    """
+    Verify Pyarrow join ids that are offset are correctly handled. 
+    """   
+    all_ids = pa.array(range(30))
+    slices = [all_ids[i * 10:i * 10 + 10]
+              for i in range(3)] 
```

Comment:
It might be easier to understand this by just writing

```
slices = [
    pa.array(range(10)),
    pa.array(range(10, 20)),
    pa.array(range(20, 30)),
]
```